binder/environment.yml

```diff
@@ -2,34 +2,34 @@ channels:
 - conda-forge
 dependencies:
 # Jupyter
-- jupyterlab=2
-- nodejs=11
-- nbconvert=5.5
-- widgetsnbextension=3.5
+- jupyterlab=3
+- nodejs=14
+- nbconvert=6
 # Python Kernel
 - ipykernel=5.1
-- ipywidgets=7.5
-- ipyleaflet=0.12
-- altair=3.1
-- bqplot=0.12
-- dask=2.1
-- matplotlib=3.1
-- pandas=0.24
+- xeus-python=0.9
+- ipywidgets=7.6
+# - ipyleaflet=0.12
+# - altair=3.1
+# - bqplot=0.12
+# - dask=2.1
+# - matplotlib=3.1
```

Comment:
Don't we want matplotlib? It's ipympl that hasn't been updated.

Reply:
Sure let add it back.